charts/drax/values.yaml

```diff
@@ -453,6 +453,20 @@ kafka:
     - name: "KAFKA_CFG_AUTO_CREATE_TOPICS_ENABLE"
       value: "true"
 
+  extraConfigYaml:
+    retention.ms: 604800000  # 7d
```

Comment:
Is this chunk of parameters different than the ones bellow that start with log. ?

Reply:
Yes, but I had misunderstood. `log.retention.ms` is a global level config, while `retention.ms` is topic level config (`log.retention.ms` would be used if `retention.ms` isn't given as part of the topic config). So it doesn't make sense here as there is no topic with the extra config...